### ISSUE TITLE
Family Protection DNS List

### ISSUE THREAD

**Issue by SMed79** [AdguardTeam/AdguardDNS#123]
Hi, 
where i can find the list which is used by the family protection DNS 176.103.130.132/134 to report unblocked adult websites ?

Thank you.

**Comment by SMed79** [comment]
Which syntax is used for the dns blocking? 
Should we report the main domain or also the subdomains?

**Comment by ameshkov** [comment]
@SMed79 here is the filter used for ad blocking:
https://github.com/AdguardTeam/AdguardDNS/blob/master/Filters/filter.txt

Also take a look at the filter template, it should give you an idea:
https://github.com/AdguardTeam/AdguardDNS/blob/master/Filters/filter.template

As Easylist maintainer, you can add domains "indirectly", just putting them into EasyList:)

**Comment by SMed79** [comment]
> As an Easylist maintainer.

No i'm not... Only **[Liste AR](https://easylist-downloads.adblockplus.org/Liste_AR.txt)** / **[Liste FR](https://easylist-downloads.adblockplus.org/liste_fr.txt)** (and [opendns](https://community.opendns.com/user/216596) domain tagging moderator).

Thank you for taking the time that you did to give me answers.

**Comment by ameshkov** [comment]
No problem:) If you want us to include anything into the filter, just create a new issue here.

> No i'm not... Only Liste AR / Liste FR (and opendns domain tagging moderator).

Btw, do you have any ad servers source in Liste AR and Liste FR?

Something like what they have in the EasyList: https://raw.githubusercontent.com/easylist/easylist/master/easylist/easylist_adservers.txt
